src/variables.js

```diff
@@ -0,0 +1,14 @@
+const AVAILABLE_CASES = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+const NO_TEXT_ERR_MESSAGE = 'Text to convert is required. '
++ 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const NO_CASE_ERR_MESSAGE = '"toCase" query param is required. '
++ 'Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".';
+const NO_AVAILABLE_CASE_ERR_MESSAGE = 'This case is not supported. '
++ 'Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.';
```

Comment:
Better to use `join` and your constant with available cases to print all available cases